pkg/apis/node/validation/validation.go

```diff
@@ -54,7 +54,8 @@ func ValidateRuntimeClassUpdate(new, old *node.RuntimeClass) field.ErrorList {
 
 func validateOverhead(overhead *node.Overhead, fldPath *field.Path) field.ErrorList {
 	// reuse the ResourceRequirements validation logic
-	return corevalidation.ValidateResourceRequirements(&core.ResourceRequirements{Limits: overhead.PodFixed}, fldPath)
+	return corevalidation.ValidateResourceRequirements(&core.ResourceRequirements{Limits: overhead.PodFixed}, fldPath,
```

Comment:
passing `false` unconditionally here has the same effect as described in https://github.com/kubernetes/kubernetes/pull/98515/files#r576984804 ... plumb PodValidationOptions into this method as well

Reply:
validateOverhead is only used for creating validation on a runtime and update validation on an existing runtime is not involved. It's ok to pass false here. @liggitt

Reply:
ah, got this mixed up with `pkg/apis/core/validation#validateOverhead`